services/frontend/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: '3.7'
+
+x-frontend: &frontend
+  build:
+    context: .
+    dockerfile: services/frontend/Dockerfile
+  image: frontend
+  volumes:
+    - ..:/govuk:delegated
```

Comment:
This is now `$GOVUK_ROOT_DIR:/govuk:delegated`

Reply:
And now it's `${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated`.  We'll get there eventually...

Reply:
Done